feeds/scheduler/poll_handler.go

```diff
@@ -0,0 +1,48 @@
+package scheduler
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type PollHandler struct {
```

Comment:
I'm not entirely sure why but having this file/struct named Poll Handler feels possibility a little confusing, on first skim I assumed it would contain the logic for polling the feeds, not accepting the external polls to the package's http server. The name does make sense, I just wonder if we could avoid the confusion?

Reply:
The justification was Handler for [http.Handler](https://golang.org/pkg/net/http/#Handler) and Poll relating to in fact polling the feeds, as FeedGroup is used to perform the actual polling operations. Open to suggestions though.

Reply:
Yeh I understand, if there wasn't other polling operations within the system then there'd be no potential confusion at all. I'm struggling to think of a suitable suggestion, but maybe something appended/prepended. Obviously this is not a blocker.

Reply:
I've gone with `FeedGroupsHandler` to be more verbose, this lines up with serving data relating to the operations of the `FeedGroups`.